src/StakeRegistry.sol

```diff
@@ -201,6 +202,8 @@ contract StakeRegistry is StakeRegistryStorage {
             nextUpdateBlockNumber: 0,
             stake: 0
         }));
+
+        quorumBitmap = uint192(quorumBitmap.plus(uint192(1) << quorumNumber));
```

Comment:
Would prefer we add a helper for this, because it's easy to fuck up. I'm using this in another branch: 

https://github.com/Layr-Labs/eigenlayer-middleware/blob/a6baa25ef060b53a46dc7c77b769d8711fbe94dc/src/libraries/BitmapUtils.sol#L303-L311